database.rules.json

```diff
@@ -1,74 +1,6 @@
 {
   "rules": {
-    "blog": {
-      ".read": true
-    },
-    "featuredSessions": {
-      "$uid": {
-        ".read": "auth.uid === $uid",
-        ".write": "auth.uid === $uid"
-      }
-    },
-    "gallery": {
-      ".read": true
-    },
-    "notifications": {
-      "subscribers": {
-        ".read": true,
-        ".write": true
-      },
-      "users": {
-        "$uid": {
-          ".read": "auth.uid === $uid",
-          ".write": "auth.uid === $uid"
-        }
-      }
-    },
-    "partners": {
-      ".read": true
-    },
-    "potentialPartners": {
-      "$id": {
-        ".write": true
-      }
-    },
-    "ratings": {
-      "$uid": {
-        ".read": "auth.uid === $uid",
-        ".write": "auth.uid === $uid"
-      }
-    },
-    "schedule": {
-      ".read": true
-    },
-    "sessions": {
-      ".read": true
-    },
-    "speakers": {
-      ".read": true
-    },
-    "previousSpeakers": {
-      ".read": true
-    },
-    "subscribers": {
-      "$id": {
-        ".write": true
-      }
-    },
-    "team": {
-      ".read": true
-    },
-    "tickets": {
-      ".read": true
-    },
-    "users": {
-      "$uid": {
-        ".read": "auth.uid === $uid",
-        ".write": "auth.uid === $uid"
-      }
-    },
-    "videos": {
-      ".read": true
-    }
+    ".read": false,
```

Comment:
remove this file we don't need it anymore

Reply:
it's better to not delete it to ensure that database is blocked for read

Reply:
it's better to not delete it to ensure that database is blocked for read